gui/src/components/onboarding/pages/mounting/ManualMounting.tsx

```diff
@@ -66,10 +69,23 @@ export function ManualMountingPage() {
     setSelectRole(BodyPart.NONE);
   };
 
+  const getCurrRotation = useCallback(
+    (role: BodyPart) => {
+      const trackers = trackerPartGrouped[role] || [];
+      if (!trackers.length || role === BodyPart.NONE) return undefined;
+      const mountingOrientation = trackers[0].tracker.info?.mountingOrientation;
```

Comment:
**NOTE:** If there happens to be more than 1 tracker assigned to a body part, the first tracker in the array will be used. Computed trackers have priority tho iirc

Reply:
**DEV-COMMENT** add filter similar to this:
https://github.com/SlimeVR/SlimeVR-Server/blob/main/server/core/src/main/java/dev/slimevr/tracking/trackers/TrackerUtils.kt (getNonInternalTrackerForBodyPosition)